torch/_C/_functorch.pyi

```diff
@@ -35,16 +35,18 @@ def _jvp_decrement_nesting() -> int: ...
 
 # Defined in aten/src/ATen/functorch/Interpreter.h
 class TransformType(Enum):
-    Torch: TransformType = ...
-    Vmap: TransformType = ...
-    Grad: TransformType = ...
-    Jvp: TransformType = ...
-    Functionalize: TransformType = ...
+    # Ignores needed to avoid mypy error 'enum members must be left unannotated'
+    Torch: TransformType = ... # type: ignore[misc]
+    Vmap: TransformType = ... # type: ignore[misc]
+    Grad: TransformType = ... # type: ignore[misc]
+    Jvp: TransformType = ... # type: ignore[misc]
+    Functionalize: TransformType = ... # type: ignore[misc]
```

Comment:
```suggestion
    # Ignores needed to avoid mypy error 'enum members must be left unannotated'
    Torch = ...
    Vmap = ...
    Grad = ...
    Jvp = ...
    Functionalize = ...
```